tests/test_e2e_05_topology.py

```diff
@@ -635,6 +635,65 @@ def test_120_removing_link_metadata_persistent(self):
         keys = data['metadata'].keys()
         assert key not in keys
 
+    def test_130_delete_link(self):
+        """Test api/kytos/topology/v3/links/{link_id} on DELETE"""
+        switch_1 = "00:00:00:00:00:00:00:01"
+        switch_2 = "00:00:00:00:00:00:00:02"
+
+        # Enable the switches and ports first
+        for i in [1, 2]:
+            sw = "00:00:00:00:00:00:00:0%d" % i
+
+            api_url = KYTOS_API + '/topology/v3/switches/%s/enable' % sw
+            response = requests.post(api_url)
+            assert response.status_code == 201, response.text
+
+            api_url = KYTOS_API + '/topology/v3/interfaces/switch/%s/enable' % sw
+            response = requests.post(api_url)
+            assert response.status_code == 200, response.text
+
+        self.restart()
+
+        # Get the link_id
+        api_url = KYTOS_API + '/topology/v3/links'
+        response = requests.get(api_url)
+        assert response.status_code == 200
+        data = response.json()
+        print("DATA -> ", data)
```

Comment:
Looks like this `print` was just temporary if you could remove it.

Reply:
Deleted